Userland/Libraries/LibGfx/ICC/BinaryWriter.cpp

```diff
@@ -623,29 +620,37 @@ static ErrorOr<Vector<ByteBuffer>> encode_tag_datas(Profile const& profile, Hash
         if (tag_data_map.contains(tag_data.ptr()))
             return {};
 
-        tag_data_bytes.append(TRY(encode_tag_data(tag_data)));
+        auto encoded_tag_data = encode_tag_data(tag_data);
+        if (!encoded_tag_data.has_value())
+            return {};
+
+        tag_data_bytes.append(TRY(encoded_tag_data.value()));
```

Comment:
and then this becomes

```cpp
auto encoded_tag_data = TRY(encode_tag_data(tag_data));
if (!encoded_tag_data.has_value())
    return {};
tag_data_bytes.append(encoded_tag_data.release_value());
```

Reply:
Done, but as said above, it's not clearly better to me, so if you have some words on why it's better it'd be appreciated :)

Reply:
(The `value()` -> `release_value()` suggestion is a good one, I understand that part.)